src/pages/Booking/SeatBooking/Seats.js

```diff
@@ -0,0 +1,197 @@
+`;
+
+const SeatsRow = styled.div`
+  grid-area: 2 / 2 / 12 / 12;
+  display: grid;
+  grid-template-columns: repeat(10, 1fr);
+  grid-template-rows: repeat(10, 1fr);
+  grid-column-gap: 0px;
+  grid-row-gap: 0px;
+`;
+
+const Seat = styled.div`
+  ${props =>
+    props.isBooked &&
+    css`
+      opacity: 0.3;
+      cursor: not-allowed;
+      pointer-events: none;
+    `}
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 20px;
+  height: 20px;
+  font-size: 8px;
+  border-radius: 3px;
+  margin: 3px 6px;
+  transition: 0.2s;
+  color: #a473e6;
+  background: ${({ theme }) => theme.colors.primary.dark};
+  cursor: pointer;
+  &:hover {
+    background: ${({ theme }) => theme.colors.primary.light};
+  }
+
+  ${props =>
+    props.isVip === true &&
+    css`
+      background: #ed9209;
+      color: #ffd08a;
+
+      &:hover {
+        background: #fcb03f;
+        box-shadow: 0 0 8px rgba(255, 255, 255, 0.7);
+      }
+    `}
+
+  ${props =>
+    props.isSelected === true &&
+    css`
+      background: #0ec78f;
+      color: #ffffff;
+      box-shadow: 0 0 8px rgba(255, 255, 255, 0.8);
+
+      &:hover {
+        background: #05a877;
+      }
+    `}
+`;
+
+const Seats = ({ values, ticketCount, bookedSeats }) => {
+  const [selectedSeats, setSelectedSeats] = useState({
+    numberOfSeats: ticketCount,
+    seatValues: [],
+  });
+
+  const { numberOfSeats, seatValues } = selectedSeats;
+
+  useEffect(() => {
+    setSelectedSeats(prev => ({
+      ...prev,
+      numberOfSeats: ticketCount,
+    }));
+
+    if (seatValues.length > ticketCount) {
+      setSelectedSeats(prev => ({
+        ...prev,
+        seatValues: [],
+      }));
+    }
+  }, [ticketCount]);
+
+  const updateSeats = useBookingStore(state => state.setSeats);
+
+  useEffect(() => {
+    updateSeats(selectedSeats);
+  }, [selectedSeats]);
+
+  const checkVipSeat = seatValue => {
+    if (
+      seatValue.startsWith('A') ||
+      seatValue.startsWith('B') ||
+      seatValue.startsWith('C')
+    ) {
+      return true;
+    } else {
+      return false;
+    }
+  };
+
+  const seatIsSelected = seat => {
+    if (seatValues.includes(seat)) {
+      return true;
+    } else {
+      return false;
+    }
+  };
+
+  const handleSelectSeat = (seat, seatCount) => {
+    if (seatValues.includes(seat)) {
+      setSelectedSeats(prev => ({
+        ...prev,
+        seatValues: [...seatValues.filter(seats => seats !== seat)],
+      }));
+      return;
+    }
+    if (seatValues.length === seatCount) {
+      setSelectedSeats(prev => ({
+        ...prev,
+        seatValues: [...seatValues.shift(), seat],
+      }));
+    }
+    setSelectedSeats(prev => ({
+      ...prev,
+      seatValues: [...seatValues, seat],
+    }));
+  };
+
+  return (
+    <SeatsGrid>
+      <SeatsAlphabet>
+        {new Array(10).fill(1).map((item, i) => (
+          <GridLabel key={i}>
+            {String.fromCharCode(item * i + 1 + 64)}
```

Comment:
```suggestion
            {String.fromCharCode(item * i + 1 + 64)}
```
1이랑 64가 의미하는게 어떤걸까요? 변수에 담아보시면 의미 전달하는게 더 좋을 것 같습니다